src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -151,7 +169,16 @@ export const ContactPolicy = InjectAppServices(
                               </a>
 
                               <span className="align-button m-l-24">
-                                <SubmitButton>{_('common.save')}</SubmitButton>
+                                <button
+                                  type="submit"
+                                  disabled={submitDisabled}
```

Comment:
Have you tried using isValid? isValid tells you if the form is valid, it is based on the validations that you pass it

Reply:
Thanks. Let's try it